functions/src/index.ts

```diff
@@ -15,6 +15,39 @@ const thingspeakUrl = (channelId: string) =>
   `https://api.thingspeak.com/channels/${channelId}/feeds.json`;
 const readingsSubcollection = (docId: string) => `/sensors/${docId}/readings`;
 
+interface pm25BufferElement {
+  timestamp: FirebaseFirestore.Timestamp | null;
+  channelAPm25: number;
+  channelBPm25: number;
+  humidity: number;
+  latitude: number;
+  longitude: number;
+}
+
+const defaultPm25BufferElement: pm25BufferElement = {
+  timestamp: null,
+  channelAPm25: NaN,
+  channelBPm25: NaN,
+  humidity: NaN,
+  latitude: NaN,
+  longitude: NaN,
+};
+
+interface AQIBufferElement {
```

Comment:
I've tended to find that style guides recomend only capitalizing the first letter of acronyms
```suggestion
interface AqiBufferElement {
```